### DISCUSSION TITLE
Retext Equality - inappropriate wording

### DISCUSSION THREAD

**Discussion by loftwah** [retextjs/retext#86] [General] [answered]
I noticed a section in the [Retext Equality rules](https://github.com/retextjs/retext-equality/blob/main/rules.md) on line 412 that implies I shouldn't use the word Indian country but that I should instead use enemy territory? This comes across as something very negative. Am I reading it wrong?

| id | type | not ok | ok |
| -- | -- | -- | -- |
| indian-country | [basic](https://github.com/retextjs/retext-equality/blob/main/rules.md#basic) | Indian country | enemy territory |

**Comment by wooorm** [comment, marked as answer]
The word Indian in the couple cases checked here are about Native Americans, as used in phrases, with an American focus.

Do you think this phrase would be used to describe the country known as India? I don't really see how that would be used in a sentence?

**Comment by loftwah** [comment]
That might be it actually. Maybe Indian country used together like that was the same as what enemy territory would be. I'm not American so I wouldn't know this context and thought it was for the country called India.

**Comment by loftwah** [comment]
That is correct. I did a search for it and it makes.sense now. Thank you for clarifying.
